api/swagger/swagger.yaml

```diff
@@ -4649,23 +4647,23 @@ paths:
         "200":
```

Comment:
Same as controller, expected response should be a 201

Reply:
Updated